ui/gl/gl_features.cc

```diff
@@ -162,6 +162,13 @@ bool UsePassthroughCommandDecoder() {
           build_info->android_build_fp(),
           kPassthroughCommandDecoderBlockListByAndroidBuildFP.Get()))
     return false;
+
+#if BUILDFLAG(USE_STARBOARD_MEDIA)
+  // TODO(crbug.com/41482978): Fix single process rendering on nvidia Shield.
+  if (build_info->manufacturer() == "NVIDIA" &&
```

Comment:
Ah, ideally we could have a new command line flag to disable those but the documentation about `kPassthroughCommandDecoderBlockListByManufacturer` etc above only refers to Field Trials (Finch), so I guess this is the next best.